app/controllers/sitemaps_controller.rb

```diff
@@ -41,7 +41,7 @@ def update
                       notice: t(".success")
         end
       else
-        format.html { render :edit }
+        redirect_to edit_planning_application_validation_tasks_path(@planning_application)
```

Comment:
is this an existing path? I just tried searching for it in the routes but couldn't find it.

Reply:
No, that was total guesswork 😭